ds-css/src/styles/custom/grid/cbp-grid.scss

```diff
@@ -0,0 +1,42 @@
+.cbp-grid {
+  display: grid;
+  gap: 16px;
+  grid-template-columns: repeat(4, 1fr);
+  height: 100vh;
+  margin: 0 20px;
+}
+
+@include tablet-portrait-view {
+  body {
+    // background-color: red;
```

Comment:
Are the temp testing of the different views safe to delete now?

Reply:
Yeap. I thought I might use it again in the future, but If I need to I'll just add it back.